tasks/install.yml

```diff
@@ -1,18 +1,72 @@
 # roles/mariadb/tasks/install.yml
 ---
-
-- name: Add official MariaDB repository
+- name: Add official MariaDB repository Centos/RedHat 7 or Fedora 30
   yum_repository:
     name: MariaDB
     description: Official MariaDB repository
     baseurl: "http://{{ mariadb_mirror }}/{{ mariadb_version }}/{{ ansible_distribution|lower|regex_replace('redhat', 'rhel')|regex_replace('oraclelinux', 'centos') }}{{ ansible_distribution_major_version }}-amd64"
     gpgkey: https://yum.mariadb.org/RPM-GPG-KEY-MariaDB
     gpgcheck: true
   tags: mariadb
+  become: true
+  when:
+    - ansible_os_family == "RedHat"
+    - ansible_facts['distribution_major_version'] == "7" or ansible_facts['distribution_major_version'] == "30"
+
+- name: Check MariaDB Centos/RegHat 8 repo installed
+  stat:
+    path: /etc/yum.repos.d/mariadb.repo
+  register: c8Mariadbpreo
 
-- name: Install packages
+- name: Install Centos/RedHat 8 MariaDB repo
+  block:
+  - name: Get MariaDB repo installer RedHat 8
+    get_url:
+      url: https://downloads.mariadb.com/MariaDB/mariadb_repo_setup
+      dest: /tmp/mariadb_repo_setup
+      mode: '0770'
+  - name: Install MariaDB repo script RedHat8
+    shell: /tmp/mariadb_repo_setup
+  - name: Remove repo installer script
+    file:
+      path: /tmp/mariadb_repo_setup
+      state: absent
+  when:
+    - not c8Mariadbpreo.stat.exists
+    - ansible_os_family == "RedHat"
+    - ansible_facts['distribution_major_version'] == "8"
+
+- name: Install Install MariaDB
   package:
     name: "{{ item }}"
     state: installed
   with_items: "{{ mariadb_packages }}"
   tags: mariadb
+
+- name: Install MySQL-python Centos/RedHat 7
```

Comment:
I think that this task and the following can now be more efficiently implemented using appropriate `vars/` files. I changed the `include_vars:` task at the beginning of `main.yml` to also allow us to specify versions.